examples/bidspath.py

```diff
@@ -0,0 +1,159 @@
+# %%
+"""
+=========================================
+An introduction to BIDSPath
+=========================================
```

Comment:
we normally don't make the === go longer than the text. Just a stylistic nitpick :)

Reply:
I've changed the title a couple times and apparently overlooked this. will adjust.

Reply:
Done in 879eaf5